lib/session/session.go

```diff
@@ -110,6 +110,46 @@ type Session struct {
 	Moderated bool `json:"moderated"`
 }
 
+// FileTransferRequestParams contain parameters for requesting a file transfer
+type FileTransferRequestParams struct {
+	// Download is true if the request is a download, false if it is an upload
+	Download bool `json:"direction"`
+	// Location is location of file to download, or where to put an upload
+	Location string `json:"location"`
+	// Filename is the name of the file to be uploaded
+	Filename string `json:"filename"`
+	// Requester is the authenticated Teleport user who requested the file transfer
+	Requester string `json:"requester"`
+	// Approvers is a list of teleport users who have approved the file transfer request
+	Approvers []Party `json:"approvers"`
+}
+
+type FileTransferResponseParams struct {
+	// RequestID is the ID of the request being responded to
+	RequestID string `json:"requestId"`
+	// Approved is true if the response approves a file transfer request
+	Approved bool `json:"approved"`
+}
+
+// UnmarshalFileTransferResponseParams takes a serialized string that contains the
```

Comment:
Don't forget to update the godoc to account for the rename (here and below)

Reply:
I don't even think these methods are needed anymore. If i'm not checking/setting any defaults and just making a new struct, I'll just do it at the callsite and remove these all together.